README.md

```diff
@@ -37,6 +37,16 @@ yarn serve
 yarn test:unit
 ```
 
+```
+yarn test:e2e
+```
+
+### Run unit tests and end-to-end tests concurrently
```

Comment:
A few things with this section. Can you add a description for just running cypress e2e tests? And, concurrently is not the right word here since the test suites don't run at the same time as a result of this command. 
```suggestion
### Run unit tests and then end-to-end tests
```

Reply:
You are absolutely right! Thank you. I will correct.